### ISSUE TITLE
Basic Activity Creates Fragments

### ISSUE THREAD

**Issue by Howlan** [google-developer-training/android-fundamentals-apps-v2#108]
**Describe the problem**
On opening a new project with Basic Activity I get two Java files 
1. MainActivity.java and 
2. FirstFragment.java 
and in res>layout four layout files
1. activity_main.xml
2. content_main.xml
3. fragment_first.xml
4. fragment_second.xml
So, I cant follow the codelab as it is based on a single java file and two layout files.

**In which lesson and step of the codelab can this issue be found?**
Lesson number 04.1 Clickable Images
Step no 3 Task 1.1 Start the new project

**How to reproduce?**
Creating a new project> Basic Activity

**Versions**
Android Studio Version 3.6.3
target API level 29

**Additional information**

**codelab:** android-fundamentals
![issue](https://user-images.githubusercontent.com/48774193/81300517-665b2e80-9095-11ea-8c30-82012875b171.jpg)

**Comment by sigjak** [comment]
Same here. Will there be an update?

**Comment by zakacat** [comment]
I was running into the same problem and I was able to go thru 4.5 RecyclerView by altering the Basic Activity on start up.
Mine also started with fragment files that I hardly know how to use and also a navigation layout which I have yet to even use.
I deleted these on start up:

-FirstFragment.java
-SecondFragment.java
-fragment_first.xml
-fragment_second.xml
-the navigation folder in res. (which contains only nav_graph.xml)
-the nav fragment layout in content_main.xml

This left me with a single activity with 2 layout files that housed a FAB button and a toolbar/appbar.

I was able to go through 4.5 with no problems.
